src/index.ts

```diff
@@ -163,7 +163,7 @@ export class UnleashClient extends TinyEmitter {
             appName,
             metricsInterval,
             disableMetrics,
-            url,
+            url: this.url.toString(),
```

Comment:
Metrics service also need to take a URL type and add the paths correctly to that, instead of doing a toString here.

Reply:
Hi @marwol-fdir, we are looking into getting this released as soon as the above comment is addressed. Thanks for contributing